engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala

```diff
@@ -64,12 +70,14 @@ class DeploymentPreparer(config: K8sDeploymentManagerConfig) extends LazyLogging
         EnvVar("SCENARIO_FILE", "/data/scenario.json"),
         EnvVar("CONFIG_FILE", "/opt/nussknacker/conf/application.conf,/data/modelConfig.conf"),
         EnvVar("DEPLOYMENT_CONFIG_FILE", "/data/deploymentConfig.conf"),
+        EnvVar("LOGBACK_FILE", "/data/logback.xml"),
         // We pass POD_NAME, because there is no option to pass only replica hash which is appended to pod name.
         // Hash will be extracted on entrypoint side.
         EnvVar("POD_NAME", FieldRef("metadata.name"))
       ),
       volumeMounts = List(
-        Volume.Mount(name = "configmap", mountPath = "/data")
+        Volume.Mount(name = "configmap", mountPath = "/data"),
+        Volume.Mount(name = "shared-configmap", mountPath = "/shared-data", readOnly = true)
```

Comment:
this mountPath look weird

Reply:
do you have any ideas what should I use instead?